docs/src/whatsnew/latest.rst

```diff
@@ -91,6 +91,9 @@ This document explains the changes made to Iris for this release
 
 #. `@bjlittle`_ added the |PyPI|_ badge to the `README.md`_. (:pull:`4004`)
 
+#. `@tkknight`_ added a banner at the top of every page if the unreleased
+   development documentatiion is being viewed. (:pull:`3999`)
+
```

Comment:
@tkknight could you add clarification to the reader that this only applies to `rtd`... that's how I understand it, right?

Reply:
Typo "if" --> "of".